Notebooks/OOPS_TMT_Data_Processing_And_QC.Rmd

```diff
@@ -102,6 +128,17 @@ Typically, one removes proteins with a single PSM, the so-called ‘one-hit wond
 psm_filt_sn_int_missing_n_features <- psm_filt_sn_int_missing %>%
   camprotR::restrict_features_per_protein(min_features=2, plot=FALSE)
 ```
+
```

Comment:
I find it helpful to have a summary of how many PSMs are retained at each stage.